network-api/networkapi/wagtailpages/migrations/0054_auto_20210325_1757.py

```diff
@@ -0,0 +1,114 @@
+# Generated by Django 2.2.17 on 2021-03-25 17:57
+
+from django.db import migrations
+import networkapi.wagtailpages.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0053_auto_20210319_2227'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='generalproductpage',
+            name='ai_is_transparent',
+            field=networkapi.wagtailpages.fields.ExtendedYesNoField(help_text='Does the company allow users to see how the AI works?'),
```

Comment:
This migration changes the ExtendedYesNoField from the buyersguide.fields.ExtendedYesNoField to wagtailpages.fields.ExtendedYesNoField. In an attempt to remove the old ExtendedYesNoField we copied it the to `wagtailpages/` app and it caused this migration. 

Because this is just a fancy CharField, we shouldn't need any tests. I did a before-and-after user test to make sure the data stayed consistent, and it did. But the more eyes on this particular change the safer it is to add.